app/src/lib/ui/base/Card.svelte

```diff
@@ -17,13 +17,11 @@
 
 <div
 	on:click={onClick}
-	class={`
-fr-card
-${horizontal ? 'fr-card--horizontal' : ''}
-${largeLink ? 'fr-enlarge-link' : ''}
-${hideArrow ? 'fr-card--no-arrow' : ''}
-${$$props.class}
-`}
+	on:keydown={onClick}
```

Comment:
il faudrait probablement in tabindex pour rendre l'élément utilisable au clavier
et valider qu'on appuie bien sur entrée